src/test/groovy/graphql/execution/AsyncExecutionStrategyTest.groovy

```diff
@@ -2,6 +2,7 @@ package graphql.execution
 
 import graphql.ErrorType
 import graphql.ExecutionResult
+import graphql.GraphQLContext
```

Comment:
Test changes: all tests instantiating a ExecutionContext ought to have had an empty or mocked GraphQLContext object added

In practice, during execution there is always an GraphQLContext inside the ExecutionContext, which is why we don't have a billion null checks for this in the engine. These tests have been brought up to date

In this file it's slightly different to the next file, because in master this file has an adaptation for the new defer feature. My intention was to keep this as close to master as possible